MiniApp/Classes/core/MiniAppSdkConfig.swift

```diff
@@ -108,5 +117,6 @@ public class MiniAppSdkConfig {
         self.analyticsConfigList = analyticsConfigList
         self.requireMiniAppSignatureVerification = requireMiniAppSignatureVerification
         self.sslKeyHash = sslKeyHash
+        self.dynamicDeepLinksList = dynamicDeepLinksList
```

Comment:
Do you think, we can pass `dynamicDeepLinksList` using `MiniApp.create` instead to keep a similar experience with `queryParams`?